I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/yui_story.rpy", line 4551, in script
    show image GoodEnding with pixellate
  File "game/yui_story.rpy", line 4551, in <module>
    show image GoodEnding with pixellate
NameError: name 'GoodEnding' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/yui_story.rpy", line 4551, in script
    show image GoodEnding with pixellate
  File "renpy/ast.py", line 1197, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "renpy/ast.py", line 1153, in show_imspec
    expression = renpy.python.py_eval(expression)
  File "renpy/python.py", line 2276, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "renpy/python.py", line 2269, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/yui_story.rpy", line 4551, in <module>
    show image GoodEnding with pixellate
NameError: name 'GoodEnding' is not defined

Windows-10-10.0.19041
Ren'Py 7.4.10.2178
Beta 1.0
Tue Dec 21 16:37:52 2021
